lib/src/disposable_manager/disposable_manager_vm.dart

```diff
@@ -14,7 +14,7 @@
 
```

Comment:
Why not just call this file `disposable_manager.dart`? Since it's used both by the VM disposable and the browser one, seems like dropping `_vm` might make sense. 🤷‍♂️